php/class-photonfill.php

```diff
@@ -580,19 +580,19 @@ public function get_attachment_image( $attachment_id, $size = 'full', $attr ) {
 				if ( 'post-thumbnail' == $size ) {
 					$size = 'full';
 				}
-				$html = '';
 				if ( photonfill_use_lazyload() ) {
 					$html = $this->get_lazyload_image( $attachment_id, $size, $attr );
 				} else {
-					$classes = $this->get_image_classes( ( empty( $attr['class'] ) ? array() : $attr['class'] ), $attachment_id, $size );
-					$html = sprintf(
-						'<img sizes="%s" srcset="%s" class="%s" %s %s>',
-						esc_attr( $this->get_responsive_image_attribute( $attachment_id, $size, 'sizes' ) ),
-						esc_attr( $this->get_responsive_image_attribute( $attachment_id, $size, 'srcset' ) ),
-						esc_attr( $classes ),
-						( ! empty( $attr['alt'] ) ) ? ' alt="' . esc_attr( $attr['alt'] ) . '"' : '',
-						( ! empty( $attr['style'] ) ) ? ' style="' . esc_attr( $attr['style'] ) . '"' : ''
+					$default_attributes = array(
+						'sizes' => $this->get_responsive_image_attribute( $attachment_id, $size, 'sizes' ),
+						'srcset' => $this->get_responsive_image_attribute( $attachment_id, $size, 'srcset' ),
 					);
+
+					$attr = wp_parse_args( $attr, $default_attributes );
+
+					$attr['class'] = $this->get_image_classes( ( empty( $attr['class'] ) ? array() : $attr['class'] ), $attachment_id, $size );
+
+					$html = wp_get_attachment_image( $attachment_id, $size, false, $attr );
```

Comment:
So the reason this doesn't use wp_get_attachment_image is because you can unset the src attribute if I recall correctly, which will force double loading of images which is bad. So you def want to keep the sprintf statments and add in attributes by iterating through the array.